src/App/components/Resource/components/Types/index.js

```diff
@@ -0,0 +1,53 @@
+import React, {PropTypes} from 'react'
+import {map, isArray} from 'lodash'
+import {Link} from 'react-router-dom'
+import colors from 'package/utils/colors'
+import Heading from 'package/components/Heading'
+
+const Arguments = ({title, types}) => (
+  <section>
+    <Heading level='2'>
+      {title}
+    </Heading>
+    <div className='flex flex-wrap justify-start'>
+      {map(types, (value, key) => (
+        <div 
+          key={key}
+          className='mb4 mr4'
+          style={{
+            minWidth: 100,
+            maxWidth: 200,
```

Comment:
I don't love these values, but they let me have container-query-esque styles with the flex + flex-wrap above; no need for media queries.